storops/unity/resource/host.py

```diff
@@ -74,6 +76,80 @@ def get_host(cls, cli, _id, force_create=False):
             ret = cls.get(cli=cli, _id=_id)
         return ret
 
+    def detach_hlu(self, lun):
+        return lun.detach_from(self)
+
+    def has_hlu(self, lun):
+        if not self.host_luns or not lun:
+            return False
+
+        has = False
+        for host_lun in self.host_luns:
+            if host_lun.lun is not None:
+                if host_lun.lun.id == lun.id:
+                    has = True
+                    break
+
+        return has
+
+    def get_hlu(self, lun):
+        if not self.host_luns or not lun:
+            return None
+
+        for host_lun in self.host_luns:
+            if host_lun.lun is not None:
+                if host_lun.lun.id == lun.id:
+                    hlu = host_lun.hlu
+                    break
+        else:
+            log.debug('Lun {} is not attached to host {}'
```

Comment:
Please use all lower case in the debug message for grep friendly.  This is a convention we took in this library.